src/analysis/souffle/examples/multimic/multimic.arcs

```diff
@@ -0,0 +1,59 @@
+// Note: This file marks speculative syntax with a SPECULATIVE comment before
+// the speculative syntax.
+
+// Represents an actual microphone within a smart mic listening to audio.
+particle MicIn
+  audio: writes AudioData { }
+// SPECULATIVE: claim a QoS integrity tag.
+//claim audio is userSpeech and is integrity#speechToTextQoS
+  claim audio is userSpeech and is speechToTextQoS
+
+// Represents the CPU within the mic.
+particle MicCompute
+  audioIn: reads AudioData { }
+  asrNeuralNet: reads writes ASRNeuralNet { }
```

Comment:
Why does this have to write also? Isn't `ASRNeuralNet` the ML model?

Reply:
It is. I thought it might be continually training on the user's speech patterns to improve the model. But if that's not the case, then writing would not be needed.

Reply:
Fixed.